source/github-helpers/index.ts

```diff
@@ -34,9 +34,38 @@ export const getCurrentCommittish = (pathname = location.pathname, title = docum
 		return parsedTitle.groups!.branch;
 	}
 
+	// .last needed for #2799 GHE #4187
+	const feedLink = select.last('link[type="application/atom+xml"]');
+	if (feedLink) {
+		return new URL(feedLink.href)
+			.pathname
+			.split('/')
+			.slice(4) // Drops the initial /user/repo/route/ part
+			.join('/')
+			.replace(/\.atom$/, '');
+	}
+
 	return unslashedCommittish;
 };
 
+export const getCurrentCommittishAnywhere = async (): Promise<string | undefined> => {
+	const committish = getCurrentCommittish();
+	if (committish) {
+		return committish;
+	}
+
+	if (!pageDetect.isRepoHome()) {
+		return;
+	}
+
+	const feedLink = await elementReady<HTMLAnchorElement>('a[data-hotkey="t"]')!;
```

Comment:
```suggestion
	const feedLink = await elementReady('a[data-hotkey="t"]')!;
```

Reply:
Need to wait for elementReady to be updated first

Reply:
https://github.com/sindresorhus/element-ready/releases